viewer/src/models/cad/distinctUntilChanged.ts

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
```

Comment:
How would you reflect that it is a typical rx utils class? (Do you want to to be a class instead of a function?)

The filename is a mishap. Fixed now.